config/default.json

```diff
@@ -10,7 +10,7 @@
     "expiresIn": 180000,
     "generateTimeout": 30000
   },
-  "SWITCH_ENDPOINT": "http://central-ledger.local:3001",
+  "ENDPOINT_SERVICE_URL": "http://central-ledger.local:3001",
```

Comment:
naming could be confusing, maybe let keep SWITCH because it is a little more descriptive